frontend/src/components/eMIB/EditEmail.jsx

```diff
@@ -1,10 +1,182 @@
 import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+
+const styles = {
+  container: {
+    maxHeight: "calc(100vh - 300px)",
+    overflow: "auto",
+    width: 700
+  },
+  header: {
+    responseTypeIcons: {
+      color: "#00565E",
+      margin: "0 8px",
+      padding: 3,
+      border: "1px solid #00565E",
+      borderRadius: 4,
+      cursor: "pointer"
+    },
+    radioButtonZone: {
+      marginBottom: 12
+    },
+    responseTypeRadio: {
+      all: "unset",
+      color: "#00565E",
+      cursor: "pointer"
+    },
+    radioPadding: {
+      marginBottom: 16
+    },
+    radioText: {
+      textDecoration: "underline"
+    },
+    textFieldBoxPadding: {
+      padding: "0 6px"
+    },
+    textField: {
+      padding: "4px 12px",
+      border: "1px solid #00565E",
+      borderRadius: 4,
+      width: "calc(100% - 36px)"
+    },
+    titleStyle: {
+      float: "left",
+      width: 24,
+      height: 32,
+      lineHeight: "2.1em"
+    }
+  },
+  response: {
+    textArea: {
+      padding: "6px 12px",
+      border: "1px solid #00565E",
+      borderRadius: 4,
+      width: "99.3%",
```

Comment:
100% for these so we don't have decimal values.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/178/commits/85f076d65301c286f95812f3b434c75b7f6ae2c6